products/identity/Sign-In Service/auth_flows/api_oauth.md

```diff
@@ -0,0 +1,187 @@
+# VA.gov API OAuth Integration
+
+## Version History
+
+| Version Number | Author(s)                                              | Revision Date | Description of Change                                                                      |
+|----------------|--------------------------------------------------------|---------------|--------------------------------------------------------------------------------------------|
+|       0.1      | Trevor Bosaw, John Bramley, Josh Scanish, Joe Niquette |    3/17/22    | Initial creation                                                                           |
+|       0.2      |                      Joe Niquette                      |    3/25/22    | Added new introspection response example, moved some sections around for better formatting |
+|       0.3      |                      John Bramley                      |    7/05/22    | Updates for mobile vs. web authentication |
+| 0.4 | John Bramley | 9/02/22 | Adds links to `vets-api` & `vets-api-mockdata` setup |
+| 0.5 | John Bramley | 9/25/23 | Updates with `ClientConfig` information |
+| 0.6 | John Bramley | 1/03/24 | `auth_flows` refactor |
+
+## Prerequisites
+
+### Postman Collection
+
+The VSP Identity team maintains a [Postman collection](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Product%20Documentation/Sign%20In%20Service/sis_postman_v1.json) to enable developers to more easily test against SiS routes; this collection is configured to manage API integrations. Documentation on how to use the SiS Postman collection can be found [here](../postman.md).
```

Comment:
It might be a good idea to spell out the acronym at the beginning of the document and then you can reference it throughout the remainder of the document. For example, "The VSP Identity team maintains a Postman collection to enable developers to more easily test against Sign in Service (SiS) routes; this collection is configured to manage API integrations. Documentation on how to use the SiS Postman collection can be found here."